bot/helper/reqHelper.ts

```diff
@@ -169,9 +170,10 @@ async function reqInfoFromAzDOAPI(
   projectName: string,
   repoName: string,
   path: string,
-  ref: string
+  ref: string,
+  accessToken?: string
 ) {
-  const token = getAzDOToken();
+  const token = accessToken ? `Bearer ${accessToken}` : getAzDOToken();
```

Comment:
What's the difference between "access token" and getAzDOToken()?

Reply:
`getAzDOToken` retrieves token string from environment variables. Removed.